diff -Nur o.qsvn-0.6.0/src/CMakeLists.txt n.qsvn-0.6.0/src/CMakeLists.txt
--- o.qsvn-0.6.0/src/CMakeLists.txt	2007-11-16 19:56:31.000000000 +0100
+++ n.qsvn-0.6.0/src/CMakeLists.txt	2007-12-28 18:40:32.364402248 +0100
@@ -21,6 +21,11 @@
 
 SET( LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib )
 
+IF ( UNIX )
+  SET( QT_QMAKE_EXECUTABLE qmake-qt4 )
+  SET( QT_MOC_EXECUTABLE moc-qt4 )
+  SET( QT_UIC_EXECUTABLE uic-qt4 )
+ENDIF ( UNIX )
 INCLUDE( ${CMAKE_ROOT}/Modules/FindQt4.cmake )
 INCLUDE( ${CMAKE_ROOT}/Modules/UseQt4.cmake )
 INCLUDE( ${CMAKE_SOURCE_DIR}/svnqt/cmakemodules/FindSubversion.cmake )
